### ISSUE TITLE
Opening geolink gives blank map

### ISSUE THREAD

**Issue by dirks** [nextcloud/maps#433]
### Steps to reproduce

- firefox is configured to open `geo:` links with nextcloud instance that has the maps app enabled
- enter geo:48.268569,5.690918 in address bar of firefox
- firefox opens https://example.com/nextcloud/index.php/apps/maps/openGeoLink/geo%3A48.268569%2C5.690918

### Expected behaviour

Map is displayed at lat, long from geo link.

### Actual behaviour

The map is not rendered

![Screenshot_20200807_132818](https://user-images.githubusercontent.com/728992/89641398-de620c00-d8b1-11ea-9d90-c5c8a6f85ed8.png)

The map will not be rendered no log event in occ.

### Server configuration

- Ubuntu 18.04
- Apache 2.4.29
- postgresql 10
- php 7.2
- nextcloud 18.0.7.1

Server is behind (apache) reverse proxy with ssl offloading.

Please let me know if you need additional information from server or client side.

**Comment by Sputnikchen** [comment]
Blank page without any working link link. No photos, no favorites, no contacts,... I've to reload the map without the geo-link (/openGeoLink/48.268569%2C5.690918) to get the map working.
The whole app is killed by the geo-link.

**Comment by downtownallday** [comment]
FWIW, on NC 25.0.3 and maps 0.2.4, I receive an Internal Server Error when following geo links.

eg: [geo:40.702179,-74.015037](geo:40.702179,-74.015037)

```
[index] Error: Exception: template file not found: template:index at <<closure>>

0. /usr/local/nextcloud/lib/private/legacy/OC_Template.php line 153
   OC\Template\TemplateFileLocator->find()
1. /usr/local/nextcloud/lib/private/legacy/OC_Template.php line 89
   OC_Template->findTemplate()
2. /usr/local/nextcloud/lib/public/AppFramework/Http/TemplateResponse.php line 207
   OC_Template->__construct()
3. /usr/local/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 178
   OCP\AppFramework\Http\TemplateResponse->render()
4. /usr/local/nextcloud/lib/private/AppFramework/App.php line 172
   OC\AppFramework\Http\Dispatcher->dispatch()
5. /usr/local/nextcloud/lib/private/Route/Router.php line 298
   OC\AppFramework\App::main()
6. /usr/local/nextcloud/lib/base.php line 1047
   OC\Route\Router->match()
7. /usr/local/nextcloud/index.php line 36
   OC::handleRequest()

GET /apps/maps/openGeoLink/geo%3A40.702179%2C-74.015037
```

**Comment by tacruc** [comment]
Should be fixed with #943
